### ISSUE TITLE
Directly set config skipping config files

### ISSUE THREAD

**Issue by nichtich** [LibreCat/Catmandu#183]
It's not documented nor recommendable how to directly set config instead of loading config files:

``` perl
use Catmandu;
my $config = get_config_from_whatever();
Catmandu->_env( $config );
```

**Comment by nichtich** [comment]
We could make `config` a setter:

~~~perl
# load from config files
use Catmandu qw(-all -load);
...
# override all configuration
config({});  # sets Catmandu->_env->config
~~~

**Comment by nics** [comment]
implemented in a828626c19469f5cbc3eaaff8852cb7e628273c9
